database/scripts/lib/buildfarm_tools.rb

```diff
@@ -71,13 +72,16 @@ def self.test_regressions_today(filter_known: false, only_consistent: false, gro
     out
   end
 
-  def self.flaky_test_regressions(filter_known: false, group_issues: false, time_range: FLAKY_BUILDS_DEFAULT_RANGE, report_regressions: [])
+  def self.flaky_test_regressions(filter_known: false, group_issues: false, time_range: FLAKY_BUILDS_DEFAULT_RANGE,
+                                  report_regressions: [])
     # Keys: job_name, build_number, error_name, build_datetime, node_name, flakiness
     out = []
     today_regressions = report_regressions
     today_regressions = test_regressions_all(filter_known: filter_known) if today_regressions.empty?
     today_regressions.each do |tr|
-      next if !tr['age'].to_i.nil? && (tr['age'].to_i >= CONSECUTIVE_THRESHOLD || tr['age'].to_i == WARNING_AGE_CONSTANT)
+      if !tr['age'].to_i.nil? && (tr['age'].to_i >= CONSECUTIVE_THRESHOLD || tr['age'].to_i == WARNING_AGE_CONSTANT)
+        next
```

Comment:
Is this expected ruby syntax? It seems more "ruby" to me as it was before with the `next if`, for some reason.